baybe/recommenders/pure/bayesian/base.py

```diff
@@ -53,17 +42,8 @@ def _get_acquisition_function_cls(
         Returns:
             The debotorchized acquisition function class.
         """
-        mapping = {
-            "PM": PosteriorMean,
-            "PI": ProbabilityOfImprovement,
-            "EI": ExpectedImprovement,
-            "UCB": partial(UpperConfidenceBound, beta=1.0),
-            "qEI": qExpectedImprovement,
-            "qPI": qProbabilityOfImprovement,
-            "qUCB": partial(qUpperConfidenceBound, beta=1.0),
-            "VarUCB": partial(UpperConfidenceBound, beta=100.0),
-            "qVarUCB": partial(qUpperConfidenceBound, beta=100.0),
-        }
+        from baybe.acquisition import debotorchize, mapping
```

Comment:
trying to undertsand why youre doing thes echange sin this file, so feel free to elaborate:

at the top of the file you still import something from botorch, hence torch. So this lazy import would have 0 impact, or am I missing something? Or should the `AcquisitionFunction` import also be lazy/type-checking only?

From another angle, this file is part of the bayesian recommender submodule. Anything bayesian (since we use botorch for it) will always have to do with torch/botorch via the acquisition functions, hence why the move here to import it lazily?

Reply:
Here's the hierarchy of imports for `base.py`:

![Screenshot 2024-03-13 at 6 18 27 PM](https://github.com/emdgroup/baybe/assets/5112312/54919cbd-b65c-4d19-9b45-6d369dd19a26)

torch only appears in the hierarchy of `baybe.utils.numericals`.

Reply:
Yes, you're right regarding the torch bubble. I'm gonna discard unrequired changes.